templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -495,6 +614,41 @@ contents:
         fi
       }
 
+      ovnk_config_dir='/etc/ovnk'
```

Comment:
I just thought that something `/var/lib/ovnk` could be less inviting for external manipulation. Not a strong opinion though so feel free to ignore.

Reply:
done